modules/fbs/fbs.info

```diff
@@ -7,6 +7,7 @@ project = fbs
 dependencies[] = date_popup
 dependencies[] = ding_provider
 dependencies[] = ding_reservation
+dependencies[] = ding_session_cache
```

Comment:
Since we add `ding_session_cache` as a dependency we also need to enable it for existing installations with a `hook_update()`.

Reply:
It's already an dependency in ding2.info, so it should be enabled.

> ding2.info:36:dependencies[] = ding_session_cache

Reply:
:+1: